lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -328,6 +328,9 @@ class MockTimers {
   #createDate() {
     kMock ??= Symbol('MockTimers');
     const NativeDateConstructor = this.#nativeDateDescriptor.value;
+    if (NativeDateConstructor.isMock) {
+      throw new ERR_INVALID_STATE('Date is already being mocked!');
```

Comment:
Nit

```suggestion
      throw new ERR_INVALID_STATE('Date is already mocked.');
```

Reply:
I'm following the same format from the other throw in the file (which also includes the `!`)

Reply:
Ah, okay. I think we don't usually shout in errors, but if that's how it is in the rest here, sure. It was a nit :)